lib/x11test.pm

```diff
@@ -607,7 +607,7 @@ sub start_clean_firefox {
     }
 
     # get rid of the reader & tracking pop-up once, first test should have milestone flag
-    $self->firefox_open_url('eu.httpbin.org/html');
+    $self->firefox_open_url('eu.httpbin.org/html', assert_loaded_url => 'firefox-urls_protocols-http');
```

Comment:
I think that at some point we should move this to an internal thing (like the html5 test page) or run an instance of it ourselves (out of the scope of this pr tho)? wdyt?

Reply:
Yes for sure